Vc/detail/neon.h

```diff
@@ -607,11 +580,8 @@ Vc_VERSIONED_NAMESPACE_END
 
 // [mask.reductions] {{{
 Vc_VERSIONED_NAMESPACE_BEGIN
-
-//NEON really doesn't have mask reduction implentation?
-
 /*
-
+//NEON really doesn't have mask reduction implentation?
```

Comment:
NEON does not have an instruction to do this directly, it was recommended to use few instructions (like vhadd)